src/Dfc.CourseDirectory.WebV2/Features/DataManagement/Venues/InProgress.cshtml

```diff
@@ -0,0 +1,66 @@
+﻿@model Dfc.CourseDirectory.Core.Models.UploadStatus
+@{
+    ViewBag.Title = "Uploading your venue data";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <h1 class="govuk-heading-xl">Uploading your venue data</h1>
+    </div>
+</div>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <div class="pttcd-loading-spinner__container">
+
+            <div class="pttcd-loading-spinner"></div>
+
+            <div class="pttcd-loading-spinner__message govuk-!-margin-4" id="StatusMessage">
+                @if (Model == Dfc.CourseDirectory.Core.Models.UploadStatus.Created)
+                {
+                    <text>Uploading file</text>
+                }
+                else
+                {
+                    <text>Processing data</text>
+                }
+            </div>
+
+            <noscript>
+                <a asp-action="InProgress" asp-all-route-data="@ProviderContext.RouteValues" id="RefreshLink">
+                    [Click to refresh]
+                </a>
+            </noscript>
+        </div>
+    </div>
+</div>
+
+@section Scripts {
```

Comment:
Inlining this for now until we find a way to generalise this, at which point it will go into a `.js` file and get bundled.